.github/workflows/e2e.yml

```diff
@@ -127,7 +133,10 @@ jobs:
       max-parallel: 1
       matrix:
         node: [16, 18]
-        npm: [8, 9]
+        npm: [8, 9, 10]
+        exclude: # node 16 is not compatible with npm 10
+          - node: 16
+            npm: 10
     runs-on: ubuntu-latest
     env:
       # The "default" npm is the one that ships with a given version of node.
```

Comment:
Does CI run with node 18.0 or latest 18? As of node 18.19.0, the default npm is now 10.

Reply:
Based on the CI, seems like its actually `v18.18.2`, should we try to pin it to a version greater than or equal to `v18.19.0`?

Reply:
v18.19.0 was only [released](https://nodejs.org/en/blog/release/v18.19.0) last week, so it seems like the tooling does use latest 18, but just hasn't pulled in the new release yet. I'd say leave keep what we have for now, but set a reminder to double check in like a week or two, maybe?